examples/nextjs/README.md

```diff
@@ -0,0 +1,133 @@
+# Use with Next.js (with new Next.js compiler)
+
+This example demonstrates how to use `jest-preview` with `Next.js`, bootstrapped using `create-next-app`.
+
+⚠️ This example is only meant for Next.js apps using the new [Next.js compiler](https://nextjs.org/docs/advanced-features/compiler) (also called Rust compiler), available from Next.js 12.0.0 onward. If you use Next.js version 11 or below, or opt out of the Next.js compiler, please see the `nextjs-babel` example.
+
+🚧 Next.js's `<Image />` component isn't working with Jest Preview yet.
+
+## Install Jest
+
+Install Jest and your favorite testing libraries, if you haven't already done so. Refer to [Next.js's official document](https://nextjs.org/docs/testing). This example uses `react-testing-library`.
+
+## Install & configure Jest Preview
+
+Install Jest Preview
+
+```bash
+npm install --save-dev jest-preview
+```
+
+Enable Jest Preview inside `jest.config.js`. Note that `enableJestPreview` is an async function that accept another async function as its parameter. The Jest config object returned by `enableJestPreview` shouldn't be modified any further.
+
+```diff
+  const nextJest = require('next/jest')
++ const { enableJestPreview } = require('jest-preview')
+
+  const createJestConfig = nextJest({
+    // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+    dir: './',
+  })
+
+  // Add any custom config to be passed to Jest
+  const customJestConfig = {
+    // Add more setup options before each test is run
+-   // setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
++   setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
```

Comment:
This is the same thing.

Reply:
I uncomment the line, since it is commented out by default according to Next.js document
<img width="723" alt="Screen Shot 2022-04-21 at 22 46 08" src="https://user-images.githubusercontent.com/28614996/164499683-97182011-1d5c-4014-bc1f-c0e58551fceb.png">

Reply:
@thanhsonng 
Oh I see. you are referring to the documentation on https://nextjs.org/docs/testing#jest-and-react-testing-library, I am looking at the examples in https://github.com/vercel/next.js/tree/canary/examples/with-jest
I think we can send them a PR to align their docs and example code.